libraries/SITL/SIM_Vicon.cpp

```diff
@@ -169,16 +172,16 @@ void Vicon::update_vicon_position_estimate(const Location &loc,
             pitch,
             yaw,
             NULL, 0);
-        msg_buf[msg_buf_index].time_send_us = time_send_us;
+        have_msg = true;
```

Comment:
I'm a little worried that this change is removing the ability to add delay into the messages.  Does it mean that we no longer use, "time_send_us"?  can this variable be removed?